Dockerfile

```diff
@@ -19,8 +19,13 @@ ENV PATH="/usr/local/bin:${PATH}"
 
 COPY scripts/* /usr/local/bin/scripts/
 COPY --from=builder /tmp/cluster-logging-operator/_output/bin/cluster-logging-operator /usr/local/bin/
+RUN mkdir /tmp/_working_dir && \
+    chmod og+w /tmp/_working_dir && \
+    yum install -y openssl
```

Comment:
@sosiouxme sanity check that we don't need to do anything for image healthiness, please?

Reply:
I don't see any new problems with the docker build (the /tmp/_working_dir thing still looks weird to me)

Reply:
That dir is used by the operator for extracting secrets into... probably doesn't need to be there so i'll add a comment to clean it up. It would also need to be addressed in code. Thanks for the sanity check